src/common/card/Card.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import styles from './card.module.scss';
+
+const Card = ({
+  width,
+  Header,
+  children,
+}: {
+  width: string;
+  Header: React.FC;
+  children: React.ReactChildren;
+}) => (
+  <div style={{ width }}>
+    <div className={styles.container}>
+      <Header />
```

Comment:
One other thing still is this `Header` I don't see a case where we have a dynamic header for the card. I would rather have a static header that takes text as a prop and render it with some styles.

Reply:
Switched the dynamic Header to a title string prop.